projects/recipe-page/phase-1-html-prompt.md

```diff
@@ -110,7 +110,7 @@ Copy the URL that appears in the search bar -- this is the URL you can  use insi
 `src` attribute of your `img` tag!
 </pre>
 
-[Here's a short video that walks you through the steps given above](https://youtu.be/lTJWBagWE4c).
+[Here's a short nineteen seconds video that walks you through the steps given above](https://youtu.be/lTJWBagWE4c).
```

Comment:
nineteen second (with no s)

Reply:
Thanks Ruthie, just made the change here is the commit hash 665efbaf8374dfd5b659ee0e0e0ae5d34fbdd7c6